content/en/docs/concepts/security/hardening-guide/authentication-mechanisms.md

```diff
@@ -0,0 +1,143 @@
+authentication for system components, such as when the Kubelet authenticates to the API Server. 
+While this mechanism can also be used for user authentication, it might not be suitable for 
+production use due to several restrictions:
+
+- Client certificates cannot be individually revoked. Once compromised, a certificate can be used 
+by an attacker until it expires. To mitigate this risk, it is recommended to configure short 
+lifetimes for user authentication credentials created using client certificates.
+- If a certificate needs to be invalidated, the certificate authority must be re-keyed, which 
+can introduce availability risks to the cluster.
+- There is no permanent record of client certificates created in the cluster. Therefore, all 
+issued certificates must be recorded if you need to keep track of them.
+- Private keys used for client certificate authentication cannot be password-protected. Anyone 
+who can read the file containing the key will be able to make use of it.
+- Using client certificate authentication requires a direct connection from the client to the 
+API server with no intervening TLS termination points, which can complicate network architectures.
+- Group data is embedded in the `O` value of the client certificate, which means the user's group 
+memberships cannot be changed for the lifetime of the certificate.
+
+## Static token file {#static-token-file}
+
+Although Kubernetes allows you to load credentials from a 
+[static token file](/docs/reference/access-authn-authz/authentication/#static-token-file) located 
+on the control plane node disks, this approach is not recommended for production servers due to 
+several reasons:
+
+- Credentials are stored in clear text on control plane node disks, which can be a security risk.
+- Changing any credential requires a restart of the API server process to take effect, which can 
+impact availability.
+- There is no mechanism available to allow users to rotate their credentials. To rotate a 
+credential, a cluster administrator must modify the token on disk and distribute it to the users.
+- There is no lockout mechanism available to prevent brute-force attacks.
+
+## Bootstrap tokens {#bootstrap-tokens}
+
+[Bootstrap tokens](/docs/reference/access-authn-authz/bootstrap-tokens/) are used for joining 
+nodes to clusters and are not recommended for user authentication due to several reasons:
+
+- They have hard-coded group memberships that are not suitable for general use, making them 
+unsuitable for authentication purposes.
+- Manually generating bootstrap tokens can lead to weak tokens that can be guessed by an attacker, 
+which can be a security risk.
+- There is no lockout mechanism available to prevent brute-force attacks, making it easier for 
+attackers to guess or crack the token.
+
+## Service account secret tokens {#service-account-secret-tokens}
```

Comment:
I'm more inclined to spell this as "ServiceAccount Secret tokens".

Reply:
cool, changed :)